### DISCUSSION TITLE
GEM Series

### DISCUSSION THREAD

**Discussion by IchDu21** [iptv-org/community#951] [Show and tell]
### Channel ID

GEMSeries.ca

### Stream URL (optional)

https://gg.hls2.xyz/live/IR%20-%20GEM%20Series%20Plus/playlist.m3u8?wmsAuthSign=c2VydmVyX3RpbWU9OC8xMy8yMDIzIDk6MzI6MDMgQU0maGFzaF92YWx1ZT1WUFZ6aWNjU00zS2NuSjFMWTdEaWVRPT0mdmFsaWRtaW51dGVzPTU=

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

**Comment by freearhey** [comment]
https://gg.hls2.xyz/live/IR%20-%20GEM%20Series%20Plus/playlist.m3u8?wmsAuthSign=c2VydmVyX3RpbWU9OC8xMy8yMDIzIDk6MzI6MDMgQU0maGFzaF92YWx1ZT1WUFZ6aWNjU00zS2NuSjFMWTdEaWVRPT0mdmFsaWRtaW51dGVzPTU= (offline)
